README.md

```diff
@@ -126,7 +126,7 @@ $HOME/gz_ws/src/asv_wave_sim/gz-waves-models/worlds
 # ensure the system plugins are found
 export GZ_SIM_SYSTEM_PLUGIN_PATH=\
 $GZ_SIM_SYSTEM_PLUGIN_PATH:\
-$HOME/gz_ws/install/lib
+$HOME/gz_ws/src/install/lib
```

Comment:
@bulentkipcak - looks like you've mistakenly built the library in the workspace `src` directory rather  than one level up. The original path is correct.

Reply:
With the exception of the gui plugin which is standalone and built in it's own folder this is a standard `colcon` workspace The build is always run in the workspace folder (`~/gz_ws`), never the `src` folder.

I won't be changing the documentation.